### PR TITLE
API version 2. See https://docs.patch.io/#/changelog for changes

### PR DESCRIPTION
**Pull request by joustrich**
### What

API version 2. See https://docs.patch.io/#/changelog for changes

### Why

To support API version 2

### SDK Release Checklist

- [x] Have you added an integration test for the changes?
- [x] Have you built the gem locally and made queries against it successfully?
- [x] Did you update the changelog?
- [x] Did you bump the package version [in the code generator](https://github.com/patch-technology/client-code-generation/blob/main/configs/ruby-config.json#L11-L12)?
- [x] If endpoints were removed, did you manually remove the corresponding files? (this should be rare)
- [x] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

**Comment by pcothenet** [comment]
@joustrich could you add a Ruby 3.1.x to the matrix. And maybe we drop 2.6.x. WDYT?

**Review by pcothenet — APPROVED** [review]
Looking good, but the issue with the spec seems legit

**Comment by joustrich** [comment]
> @joustrich could you add a Ruby 3.1.x to the test matrix? And maybe we drop 2.6.x. WDYT?

Sounds good to me!

File: spec/api/order_line_items_api_spec.rb [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  # @param order_id 
+  # @param create_order_line_item_request 
+  # @param [Hash] opts the optional parameters
+  # @option opts [Integer] :patch_version 
+  # @return [OrderLineItemResponse]
+  describe 'create_order_line_item test' do
+    it 'should work' do
+      # assertion here. ref: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers
+    end
+  end
```
Comment (rmody3) [inline spec/api/order_line_items_api_spec.rb:47]: just confirming we will want specs here correct?

**Review by rmody3 — APPROVED** [review]

**Pull request merged**